commands/apps_test.go

```diff
@@ -45,3 +47,137 @@ func TestCommandsAppsCreateAttemptsToValidateStackOnError(t *testing.T) {
 	assert.Error(err)
 	assert.Regexp("bad request: unable to find stack", err)
 }
+
+// Looks like nested functions are not yet supported by the compiler
+func OverwriteFile(loc string, data string) (err error) {
+	err = os.Remove(loc)
+	if err != nil {
+		log.Println("[ERROR] unable to remove files, perhaps they do not exist?")
+	}
+	f, err := os.Create(loc)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	_, err = f.WriteString(data)
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func TestInitNodejsBasicAppEncounteringPossibleFailureStates(t *testing.T) {
```

Comment:
Great to see some tests! 

The test name should: 

- namespaced (`TestCommandsAppsInit`, so you can run a subset of tests with `go test ./... -run TestCommandsAppsInit`), and
- more direct

How about `TestCommandsAppsInitHandlesErrors`?

Reply:
namespace like the scoping in languages like c++ or would you just like me to change the name?

Reply:
Changing the name is perfect. 

That will make this test consistent with the rest of the code base.